tests/em/fdem/forward/test_FDEM_analytics.py

```diff
@@ -61,8 +61,8 @@ def setUp(self):
         prb.pair(survey)
 
         try:
-            from pymatsolver import PardisoSolver
-            prb.Solver = PardisoSolver
+            from pymatsolver import Pardiso
```

Comment:
Other people do not have to update the solver, the interface did not change so anyone can still use an older version of `pymatsolver`. I would like to keep the tests up to date with the other pieces though.

People can update their versions of pymatsolver when they are ready. In another PR I would like to address #532, which would require updating of pymatsolver.